vars/aws.example.yml

```diff
@@ -6,7 +6,7 @@ cluster_name: openshift
 
 # The OpenShift version to install
 # IMPORTANT: Make sure this value is quoted, otherwise it gets read as 3.1 instead of 3.10
-openshift_version: "3.10"
+openshift_version: "3.11.43"
```

Comment:
The value for `openshift_version` must be `3.xx` and doesn't support the format `3.xx.xx`.

To set a z-stream release, there is a separate variable named `openshift_version_minor`.

In this example, you'd set the following:

```yaml
openshift_version: "3.11"
openshift_version_minor: "3.11.43"
```